src/Cursor.js

```diff
@@ -37,8 +37,10 @@ function update(cmp, path, operation, nextValue) {
   var nextState;
 
   if (path.length > 0) {
+    var q = cmp._reactInternalInstance._pendingStateQueue;
```

Comment:
@dustingetz I share this concern. I don't know that there is an api to access pendingState. I think this is a lateral change as we were illegally grabbing cmp._pendingState before.